docs_src/pipelinerun.md

```diff
@@ -1,23 +1,25 @@
 # Pipeline Run
 
 ## Prerequisites: 
-Pipeline setup needs to be done first, pipeline setup documentation be found [HERE](./pipelinesetup.md)
+Pipeline setup needs to be done first, pipeline setup documentation be found [HERE](./pipelinesetup.md).
 
 ---
 ## Introduction to Pipeline run flow and expectation
-1. Pipeline can be using differnt media type as input choice:
+The pipeline run is meant to bring the service up and perform inferencing on the selected input media. The output of the pipeline run provides the inference results for each frame regarding the media source such as text, barcode, etc; as well as the FPS (frames per second). Pipeline run provides many options in media type, system process platform type, and additional optional parameters. These options give you the oppotunity to compare what system process platform is better for your need. There are few choices you have to make as inputs to the pipeline run script:
+
+1. Pipeline can be run using different media type as input choice:
     - pipeline run with camera simulator using rtsf
     - pipeline run with realsense camera
     - pipeline run with USB camera
     - pipeline run with a video file
-2. Pipeline can be using different platform type as input choice:
+2. Pipeline can be run using different platform type as input choice:
     - core
     - gpu.0
     - gpu.1
     - xeon
 3. Pipeline can be run with more optional parameters, see [Optional parameters](#optional-parameters) section for detail.
 
```

Comment:
maybe another sub header (or bolded text here) to say `Pipeline Process` or something like that.